ws-nextjs-app/pages/[service]/live/[id]/Post/index.test.tsx

```diff
@@ -151,6 +156,21 @@ describe('Post', () => {
       ).toBeInTheDocument();
     });
 
+    it('should render the new media player in a post containing audio', async () => {
+      const { container } = await act(async () => {
+        return render(<Post post={audioSamplePost} />, {
+          id: 'c7p765ynk9qt',
+          service: 'pidgin',
+          pageType: LIVE_PAGE,
+          pathname: '/pidgin/live/c7p765ynk9qt',
+        });
+      });
+
+      expect(
+        container.querySelector('[data-e2e="media-loader__placeholder"]'),
+      ).toBeInTheDocument();
```

Comment:
Maybe we need to be more explicit in this test to check the audio SVG is in the page?

Reply:
This now checks for "data-e2e="media-loader" as theres no longer a placeholder for audio - theres no audio SVG to check for anymore either, is specific enough do you think ?

Reply:
Yeah it's better, I guess it depends what your trying to test here, are you actually testing for Audio, or are you testing that if there's an audio block it just needs to render the media loader